server/service/client_scripts.go

```diff
@@ -57,6 +57,10 @@ func (c *Client) runHostScript(verb, path string, hostID uint, scriptContents []
 			return nil, errors.New(fleet.RunScriptScriptsDisabledGloballyErrMsg)
 		}
 		return nil, errors.New(fleet.RunScriptForbiddenErrMsg)
+	case http.StatusGatewayTimeout:
```

Comment:
Do we ever return that status code? It looks like we use `http.StatusRequestTimeout` instead: https://github.com/fleetdm/fleet/blob/4197076b1697fb1374d40d9237bc8564b030a1f3/server/service/scripts.go#L77-L86

Reply:
Ahh nevermind, I see that the 504 is actually from nginx: https://github.com/fleetdm/fleet/issues/19378#issuecomment-2152664446

Reply:
(might be worth adding a comment explaining this, as it would be tempting to remove the code if someone sees that we never return 504 ourselves)